pkg/component/autoscaling/clusterautoscaler/cluster_autoscaler.go

```diff
@@ -176,6 +186,23 @@ func (c *clusterAutoscaler) Deploy(ctx context.Context) error {
 		return err
 	}
 
+	if c.HasExpanderConfigMapConfigured() {
+		configMap := &corev1.ConfigMap{}
+		if err := c.gardenClient.Get(ctx, client.ObjectKey{Name: *c.config.ExpanderConfig.ConfigMapName, Namespace: c.projectNamespace}, configMap); err != nil {
+			return err
+		}
+
+		configMap.Namespace = c.namespace
+		configMap.Name = ClusterAutoScalerPriorityConfigMapName
+
+		if _, err := controllerutils.GetAndCreateOrMergePatch(ctx, c.client, configMap, func() error {
```

Comment:
Initially, that was the idea I think, yes.
I think it could also be a good feature to let shoot owners maintain this config via the specified `ConfigMap` in the virtual-garden cluster.
Although I can imagine that this could be confusing / not wanted by cluster owners to configure this outside of the actual cluster.
If we decide to just copy the contents once during creation, I propose to rename the field in the `Shoot` spec to something like `initConfigMap` and update the documentation accordingly.
What do you think?

Reply:
Yes, let's rename the field (proposal: `initialExpanderConfig`) and make it immutable. This way, we don't break the flow/"API" of existing users.

Reply:
Yup, sounds good to me :)